src/app/content/highlights/reducer.ts

```diff
@@ -9,13 +9,20 @@ import * as actions from './actions';
 import { highlightingFeatureFlag } from './constants';
 import { State } from './types';
 
+const defaultColors = [
+  HighlightColorEnum.Blue,
+  HighlightColorEnum.Green,
+  HighlightColorEnum.Pink,
+  HighlightColorEnum.Purple,
+  HighlightColorEnum.Yellow,
+];
```

Comment:
you could derive this from the constant like you did in ColorFilter instead of hardcoding it